### DISCUSSION TITLE
Include Automatic Irrigation Valve

### DISCUSSION THREAD

**Discussion by mengi69** [Koenkk/zigbee2mqtt#12469] [Devices]
Hello

I have a Zigbee product that I would have liked to use through Zigbee2mqtt

Zigbee model = TS0601

Zigbee Manufacturer = _TZE200_arge1ptm

Tuya - Irrigation valve

Thanks in advance

![111](https://user-images.githubusercontent.com/4528723/168341611-d294c87b-faa8-48dc-a4c9-5c9168dc9b23.jpg)

![Valvula](https://user-images.githubusercontent.com/4528723/168341674-d877356f-36e6-443a-a8a3-7bdcbc0cba40.jpeg)

**Comment by mengi69** [comment]
Hello again.

Does anyone know how they can help me?

Thank you

**Comment by dedwards66** [comment]
Over on the ZHA github there is a discussion that it making progress on getting this working.
https://github.com/zigpy/zha-device-handlers/issues/1551

**Comment by mengi69** [comment]
Thank you.

I'm going to follow her.

**Reply by mathgoy** [reply]
Hi,

I have a similar valve. 
Did you manage to make it work?

**Reply by mengi69** [reply]
Hello.
No, I'm following the thread that was passed to me above, which is making some progress.
